ibm/resource_ibm_lb_vpx.go

```diff
@@ -528,6 +533,7 @@ func resourceIBMLbVpxRead(d *schema.ResourceData, meta interface{}) error {
 
 	d.Set("vip_pool", vips)
 	d.Set("ip_count", ipCount)
+	d.Set("management_ip_address", *getObjectResult.ManagementIpAddress)
```

Comment:
I have found that `Set` API takes care of the pointer indirection. Though at places you would still see the explicit way of doing as you have done before, but in newer codes I think it makes sense to leverage that. So you could just say `d.Set("management_ip_address", getObjectResult.ManagementIpAddress)`. This has the added benefit that if pointer is nil it won't blow even if you have not checked the `nilness` as the `Set` takes care of that.